pkg/obfuscate/obfuscate.go

```diff
@@ -105,7 +105,10 @@ type StatsClient interface {
 type SQLConfig struct {
 	// TableNames specifies whether the obfuscator should also extract the table names that a query addresses,
 	// in addition to obfuscating.
-	TableNames bool
+	TableNames bool `json:"table_names"`
+
+	// CollectComments specifies whether the obfuscator should also extract comments from the SQL statement.
```

Comment:
```suggestion
	// CollectComments specifies whether the obfuscator should extract and return comments as SQL metadata when obfuscating.
```
Let's "hint" at the SQLMetadata struct.